ts/screens/wallet/ConfirmCardDetailsScreen.tsx

```diff
@@ -134,17 +142,44 @@ class ConfirmCardDetailsScreen extends React.Component<Props, State> {
       title: I18n.t("global.buttons.back")
     };
 
-    return (
-      <BaseScreenComponent
-        goBack={true}
-        headerTitle={
-          isInPayment
-            ? I18n.t("wallet.saveCardInPayment.header")
-            : I18n.t("wallet.saveCard.header")
-        }
-        contextualHelpMarkdown={contextualHelpMarkdown}
-        faqCategories={["wallet_methods"]}
-      >
+    // shown when wallets pot is in error state
+    const walletsInErrorContent = (
+      <SafeAreaView style={bonusVacanzeStyle.flex}>
```

Comment:
Maybe it's better to use `IOStyles.flex` which is a global style value, the one you used is more feature scoped, what do you think?

Reply:
update within f891368